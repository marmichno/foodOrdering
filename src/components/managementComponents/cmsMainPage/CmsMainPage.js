import {CmsNavbar} from '../cmsNavbar/CmsNavbar';

export const CmsMainPage = () =>{
    return(
        <div className="cmsMainPageContainer">
            <CmsNavbar/>
            <div className="cmsMainPageContent">
                <h1>There will be graphed based on sales data</h1>
            </div>
        </div>
    )
}