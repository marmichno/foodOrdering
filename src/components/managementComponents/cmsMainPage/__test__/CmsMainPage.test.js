import {CmsMainPage} from '../CmsMainPage';
import TestRenderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";
import { Provider } from 'react-redux';
import { createStore, applyMiddleware } from 'redux';


const store = createStore(() => [], {}, applyMiddleware());

test("matches snapshot", () => {
    const tree = TestRenderer.create(<Provider store={store}><CmsMainPage/></Provider>).toJSON();
    expect(tree).toMatchSnapshot();
});