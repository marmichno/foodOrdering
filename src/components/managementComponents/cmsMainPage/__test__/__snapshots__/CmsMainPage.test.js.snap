// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="cmsMainPageContainer"
>
  <div
    className="hamburgerMenu"
  >
    <svg
      fill="none"
      height="1em"
      onClick={[Function]}
      stroke="currentColor"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeWidth="2"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 24 24"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <line
        x1="3"
        x2="21"
        y1="12"
        y2="12"
      />
      <line
        x1="3"
        x2="21"
        y1="6"
        y2="6"
      />
      <line
        x1="3"
        x2="21"
        y1="18"
        y2="18"
      />
    </svg>
  </div>
  <div
    className="cmsMainPageContent"
  >
    <h1>
      There will be graphed based on sales data
    </h1>
  </div>
</div>
`;
